src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/UserEntity.kt

```diff
@@ -41,7 +41,8 @@ data class UserEntity(
   val id: UUID,
   var name: String,
   val deliusUsername: String,
-  val deliusStaffIdentifier: Long,
+  var deliusStaffCode: String?,
```

Comment:
so is it might worth saying explicitly in the commit message and PR that once all pre-existing records have been updated we intend to make this a mandatory property?

And does that make the case for some kind of data migration process to attempt to update all at the same time rather than having to wait an undefined period for users to log in and be re-fetched individually? 

If I'm understanding properly, that is?

Reply:
> And does that make the case for some kind of data migration process to attempt to update all at the same time rather than having to wait an undefined period for users to log in and be re-fetched individually?

Agreed, will do this in a subsequent PR as will need to do some scaffolding similar to seeding.  Will update the commit message to reflect this.